### PR TITLE
Feature/add outtake subsystem

### LINKED ISSUE #18: Create Outtake subsystem
Create the outtake subsystem. 

Should be driven by 1 motor. Method needed for setting speed.

When initialized (and always) motor should be in Brake mode.

### PR DESCRIPTION
**Pull request by chit-sudeep**
Creates the Outtake subsystem. Includes a method to set speed of the outtake motor.

Closes #18

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Outtake.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    outtakeMotor = new CANSparkMax(outtakeId, motorType);
+    outtakeMotor.setIdleMode(IdleMode.kBrake);
+  }
+
+  /**
+   * Sets the Outtake {@link CANSparkMax} motor with a given speed.
+   *
+   * @param outtakeSpeed The speed to set the {@link CANSparkMax} motor to.
+   */
+  public void setOuttakeSpeed(double outtakeSpeed) {
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/subsystems/Outtake.java]: Naming is slightly redundant. Change to this
```suggestion
  public void setSpeed(double speed) {
```

**Review by aidan-mundy — CHANGES_REQUESTED** [review]
Pretty good, just a minor naming change to fix. Let me know when this has been implemented and I'll approve.

**Review by aidan-mundy — CHANGES_REQUESTED** [review]
Pretty good, just a minor naming change to fix. Let me know when this has been implemented and I'll approve.

File: src/main/java/frc/robot/subsystems/Outtake.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   */
+  public Outtake(int outtakeId, MotorType motorType) {
+    outtakeMotor = new CANSparkMax(outtakeId, motorType);
+    outtakeMotor.setIdleMode(IdleMode.kBrake);
+  }
+
+  /**
+   * Sets the Outtake {@link CANSparkMax} motor with a given speed.
+   *
+   * @param outtakeSpeed The speed to set the {@link CANSparkMax} motor to.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/subsystems/Outtake.java]: Need to update the corresponding Javadocs after updating a method

**Review by aidan-mundy — CHANGES_REQUESTED** [review]

**Review by aidan-mundy — APPROVED** [review]

**Review by kchenturtles — APPROVED** [review]
Read through the code + Javadocs. It looks good!

**Pull request merged**